osquery/tables/applications/chrome/chrome_extensions.cpp

```diff
@@ -75,25 +75,24 @@ QueryData genChromeExtensions(QueryContext& context) {
       row["from_webstore"] = SQL_TEXT(
           getExtensionProfileSettingsValue(extension, "from_webstore"));
 
+      // Raw install time
       row["install_time"] =
           SQL_TEXT(getExtensionProfileSettingsValue(extension, "install_time"));
 
-      std::int64_t converted_timestamp{-1};
-
-      auto converted_timestamp_exp =
-          webkitTimeToUnixTimestamp(row.at("install_time"));
-
-      if (converted_timestamp_exp.isError()) {
-        LOG(ERROR) << "Failed to parse the install_time value '"
-                   << row.at("install_time")
-                   << "' from the following extension: " << extension.path;
-
-      } else {
-        converted_timestamp = converted_timestamp_exp.take();
+      // If the install time is present, and we can convert it, do so.
+      if (row["install_time"] != "") {
```

Comment:
Prefer to check .empty() instead of comparing to a blank string.

Reply:
Okay!